src/client/components/MessageList/MessageList.js

```diff
@@ -3,22 +3,24 @@
 import React from 'react';
 import Message from '../Message/Message';
 import PropTypes from 'prop-types';
-import '../Message/Message.css';
+import './MessageList.css';
```

Comment:
Nice!! This is a great change. Shows that you have an eye for detail. That it was not the correct file and then fixing that